known-issues/servers-in-cluster.md

```diff
@@ -0,0 +1,26 @@
+---
+title: SQL logs are not being collected for servers in a SQL cluster.
+description: SQL logs are not being collected for servers in a Failover Cluster or AlwaysOn Availability Group.
+author: Tomislav Sirovec
+date: 28/06/2018
+---
+
+__Summary:__  
+The problem appears when trying to collect SQL logs from servers that are a member of a Failover Cluster or AlwaysOn Availability Group. In this case, out of the box collection of SQL logs is currently not supported.  
+But, there is a workaround. 
+
+__Solution:__  
+The problem is that, by default, the application does not see individual servers (nodes) in a cluster.  
```

Comment:
maybe modify to -- "The problem is that application cannot currently automatically detect all servers (nodes) that are part of that Cluster."